boards/arm/nrf52840_pca10056/board.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <board.h>
+#include <nrf_power.h>
+
+static int board_nrf52840_pca10056_init(struct device *dev)
```

Comment:
This is more generic functionality - should belong to SoC instead of the board. Awaiting for @anangl comment where to put that.

Reply:
@anangl had you already the chance to look about this?

Reply:
@nvlsianpu has a point.
If this snippet is taken from nrf52840_pca10059 board.c then it would make even more sense to put it in a soc.c file so that it is maintained in one place.